libraries/wallet/include/graphene/wallet/wallet.hpp

```diff
@@ -632,6 +632,12 @@ class wallet_api
        */
       bool    load_wallet_file(string wallet_filename = "");
 
+      /** Exits from Graphene wallet.
+       *
+       * The current wallet will be closed.
+       */
+      void    exit_wallet();
+
```

Comment:
please just name this function `exit` or `quit`. the user already is inside the wallet.

Reply:
quit name was choosed by me